data/src/main/java/camp/nextstep/edu/github/data/DataToDomainMapper.kt

```diff
@@ -0,0 +1,5 @@
+package camp.nextstep.edu.github.data
+
+interface DataToDomainMapper<T> {
```

Comment:
```suggestion
internal interface DataToDomainMapper<T> {
```
data 모듈에서만 사용하는 인터페이스이므로, internal 변경자를 붙여보면 어떨까요?